### ISSUE TITLE
Bucketing in MXNet.jl

### ISSUE THREAD

**Issue by vchuravy** [dmlc/MXNet.jl#129]
I think we are missing the infrastructure necessary to implement bucketing? http://mxnet.readthedocs.io/en/latest/how_to/bucketing.html

@ranjanan This would be helpful for you.

**Comment by pluskid** [comment]
@vchuravy Yes, we could add that in the current FeedForward interface. Or it would be better if we could add the whole Module API in the Python side to Julia. I have wanted to do this, but never found a chunk of continuous time slot to work on it.

**Comment by vchuravy** [comment]
You mean http://mxnet.readthedocs.io/en/latest/packages/python/module.html ? I can make an attempt at wrapping the underlying interface.

**Comment by pluskid** [comment]
@vchuravy Yes, exactly! You are very welcome to if you have time!

**Comment by ZacCranko** [comment]
Any progress on wrapping the Module interface?
